docs/notebooks/active_learning.pct.py

```diff
@@ -0,0 +1,177 @@
+# %% [markdown]
+# # Active Learning
+
+# %% [markdown]
+# Sometimes rather than optimizing a black-box function, we want to learn the function better by reducing our model uncertainty. The one way to do this is by doing sampling in unknown inputs regions (active learning). This notebook demonstrates to perform Bayesian active learning using `trieste`.
```

Comment:
Sometimes, we may just want to learn a black-box function, rather than optimizing it.  This goal is known as active learning and corresponds to choosing query points that reduce our model uncertainty. This notebook demonstrates to perform Bayesian active learning using `trieste`.